components/molecule/select/src/components/SingleSelection.js

```diff
@@ -30,7 +30,9 @@ const MoleculeSelectSingleSelection = props => {
   const handleSelection = (ev, {value}) => {
     onChange(ev, {value})
     onToggle(ev, {isOpen: false})
-    refMoleculeSelect.current.focus()
+    refMoleculeSelect &&
```

Comment:
`refMoleculeSelect?.current?.focus()`

Reply:
I know but the linter is crashing with the optional chaining

Reply:
![image](https://user-images.githubusercontent.com/1105226/73258840-d4ef3d00-41c6-11ea-8f0e-886c6ce85a77.png)

Reply:
have you tried to turn the computer on and off?

Reply:
... perhaps you need to reinstall node_modules 😃

Reply:
🥇 it was the first thing I did